File: Stats.CPP/Basic_Chart.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Basic_Chart.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "Basic_Form"
#pragma resource "*.dfm"
TfrmBasicChart *frmBasicChart;
//---------------------------------------------------------------------------
__fastcall TfrmBasicChart::TfrmBasicChart(TComponent* Owner)
	: TfrmBasic(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TfrmBasicChart::Button1Click(TObject *Sender)
{
	EditChart(this,Chart1);
}
//---------------------------------------------------------------------------

File: Stats.Delphi/Basic_Chart.pas
unit Basic_Chart;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
  Basic_Form, StdCtrls, ExtCtrls, TeeProcs, TeEngine, Chart, TeeTools,
  TeeGDIPlus;

type
  TfrmBasicChart = class(TfrmBasic)
    Chart1: TChart;
    Panel2: TPanel;
    Button1: TButton;
    procedure Button1Click(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  frmBasicChart: TfrmBasicChart;

implementation

Uses EditChar;
{$R *.DFM}

procedure TfrmBasicChart.Button1Click(Sender: TObject);
begin
  EditChart(Self,Chart1);
end;

end.
